frontend/src/components/LendingForm.js

```diff
@@ -1,180 +1,201 @@
-import React, {useCallback, useEffect, useState} from 'react';
-import {useNavigate} from 'react-router-dom';
-import {getTokenBalances, sendTransaction} from '../utils/wallet';
-
-const LendingForm = ({walletId}) => {
-    const navigate = useNavigate();
-    const [balances, setBalances] = useState({});
-    const [formData, setFormData] = useState({
-        token: '',
-        amount: '',
-        multiplier: ''
-    });
-    const [transactionData, setTransactionData] = useState(null);
-    const [transactionStatus, setTransactionStatus] = useState(null);
-    const [isLoading, setIsLoading] = useState(false);
-
-    const fetchBalances = useCallback(async () => {
-        if (!walletId) return;
-        try {
-            const tokenBalances = await getTokenBalances(walletId);
-            setBalances(tokenBalances);
-        } catch (error) {
-            console.error("Failed to fetch balances:", error);
-            alert("Failed to fetch token balances. Please try again.");
+import React, { useCallback, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { getTokenBalances, sendTransaction } from 'utils/wallet';
+
+const LendingForm = ({ walletId }) => {
+  const navigate = useNavigate();
+  const [balances, setBalances] = useState({});
+  const [formData, setFormData] = useState({
+    token: '',
+    amount: '',
+    multiplier: '',
+  });
+  const [transactionData, setTransactionData] = useState(null);
+  const [transactionStatus, setTransactionStatus] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const fetchBalances = useCallback(async () => {
+    if (!walletId) {
+      return;
+    }
+    try {
+      const tokenBalances = await getTokenBalances(walletId);
+      setBalances(tokenBalances);
+    } catch (error) {
+      console.error('Failed to fetch balances:', error);
+      alert('Failed to fetch token balances. Please try again.');
```

Comment:
change to notify

Reply:
It is changed to notify. Please check it out. Thanks.